work6.py

```diff
@@ -0,0 +1,23 @@
+# Задача 6: Вы пользуетесь общественным транспортом?
+# Вероятно, вы расплачивались за проезд и получали билет с номером.
+# Счастливым билетом называют такой билет с шестизначным номером,
+# где сумма первых трех цифр равна сумме последних трех.
+# Т.е. билет с номером 385916 – счастливый,
+# т.к. 3+8+5=9+1+6. Вам требуется написать программу, которая проверяет счастливость билета.
+
+
+print('Введите шестизначный номер билета')
+a = (input())
+
+if len(a) != 6:
+    print('Число не шестизначное')
+else:
+    b = list(map(int, list(a)))
```

Comment:
Переводить строку в список на этом этапе не нужно.
```suggestion
    b = list(map(int, a))
```